QModManager/OptionsManager.cs

```diff
@@ -1,36 +1,36 @@
 ﻿namespace QModManager
 {
     using Harmony;
-    using QModManager.API.SMLHelper.Utility;
+    //using QModManager.API.SMLHelper.Utility;
     using UnityEngine.Events;
 
     internal static class OptionsManager
     {
         internal static int ModsTab;
 
-        [HarmonyPatch(typeof(uGUI_OptionsPanel), "AddTabs")]
-        private static class OptionsPatch
-        {
-            [HarmonyPostfix]
-            private static void Postfix(uGUI_OptionsPanel __instance)
-            {
-                ModsTab = __instance.AddTab("Mods");
-                __instance.AddHeading(ModsTab, "QModManager");
+        //[HarmonyPatch(typeof(uGUI_OptionsPanel), "AddTabs")]
+        //private static class OptionsPatch
+        //{
+        //    [HarmonyPostfix]
+        //    private static void Postfix(uGUI_OptionsPanel __instance)
+        //    {
+        //        ModsTab = __instance.AddTab("Mods");
+        //        __instance.AddHeading(ModsTab, "QModManager");
```

Comment:
Why did you delete this?

Reply:
likely a mistake from trying to get the solution building again